docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/import-eks-cluster-enable-hybrid-mode.md

```diff
@@ -0,0 +1,517 @@
+
+    ```yaml {13-17} hideClipboard
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: aws-auth
+      namespace: kube-system
+    data:
+      mapRoles: |
+        - groups:
+          - system:bootstrappers
+          - system:nodes
+          rolearn: arn:aws:iam::111122223333:role/eks-nodegroup-example
+          username: system:node:{{EC2PrivateDNSName}}
+        - groups:
+          - system:bootstrappers
+          - system:nodes
+          rolearn: <roleArn>
+          username: system:node:{{SessionName}}
+    ```
+
+18. If you need to configure proxies for edge host communication or you plan to specify a VPN server IP for one or more
+    edge hosts, create a ConfigMap using the following template.
+
+    You can omit the `serviceCIDR` or `vpcCIDR` entries if they are not required.
+
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: hybrid-config
+      namespace: kube-system
+    data:
+      serviceCIDR: "<serviceCidrIp>" # Only required if using a proxy configuration for edge hosts. If not specified, defaults to 10.96.0.0/12.
+      vpcCIDR: "<vpcCidrIp>" # Only required if a VPN server IP is configured for one or more edge hosts. No default value.
+    ```
+
+19. Replace `<serviceCidrIp>` with your hybrid pod CIDR list. For example, `192.168.0.0/16`. The `serviceCIDR`
+    configures `NO_PROXY` to prevent Kubernetes service traffic from routing through the proxy server.
+
+20. Replace `<vpcCidrIp>` with your hybrid node CIDR list. For example, `10.200.0.0/16`. The `vpcCIDR` defines the
+    network range for VPC resources to ensure proper routing through the VPN.
+
+21. Apply the ConfigMap to your cluster.
+
+    Example command.
+
+    ```shell hideClipboard
+    kubectl create -f=/dev/stdin <<-EOF
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: hybrid-config
+      namespace: kube-system
+    data:
+      serviceCIDR: "192.168.0.0/16"
+      vpcCIDR: "10.200.0.0/16"
+    EOF
+    ```
+
+### Validate
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. Navigate to the left **Main Menu** and select **Clusters**.
+
+3. Click on the cluster you created to view its details in the **Overview** tab.
+
+4. Ensure the **Cluster Status** field displays **Running**.
+
+## Configure CNI for Hybrid Nodes
+
+You need to configure a CNI for Amazon EKS Hybrid Nodes as the standard Amazon EKS VPC CNI lacks hybrid node support.
+Cilium handles IP Address Management (IPAM) and Border Gateway Protocol (BGP) for non-AWS nodes.
+
+### Prerequisites
+
+- Your Palette account role must have the `clusterProfile.create` permission to import a cluster profile. Refer to the
+  [Cluster Profile](../../../../user-management/palette-rbac/project-scope-roles-permissions.md#cluster-profile)
+  permissions for guidance.
+
+If enabling [Cilium Envoy](https://docs.cilium.io/en/latest/security/network/proxy/envoy/) or other Cilium add-ons, you
```

Comment:
Not sure if this one is supposed to be nested? But it looks a bit strange visually going from list, no list, list again. That might just be me.

Reply:
I'll adjust this to make it clearer.